web/channels/retro_channel.ex

```diff
@@ -45,6 +45,22 @@ defmodule RemoteRetro.RetroChannel do
     {:noreply, socket}
   end
 
+  def handle_in("new_idea", %{"body" => body, "category" => "action-item", "userId" => user_id, "assigneeId" => assignee_id}, socket) do
```

Comment:
hmmm... this handler is nearly identical to the handler defined below. I wonder if we can just delete it outright and pass a `null` `assigneeId` in cases where. The other alternative would be to extract the commonalities between this handler and the one above out into a private function, to avoid the repetition.

Reply:
lemmeknow whatcha think!

Reply:
Just pushed up a commit that does just that.

Reply:
Or, made one. Guess I haven't pushed it yet.